src/views/LimitedRestorationExtension.vue

```diff
@@ -335,10 +334,8 @@ export default class LimitedRestorationExtension extends Mixins(
       throw new Error(`Applicant not found for ${this.getBusinessId}`)
     }
 
-    // make a copy of the original object and assign a new id (for UI use only)
+    // make a copy of the original object
     const copy = cloneDeep(applicant)
-    copy.officer.id = uuidv4()
-
```

Comment:
Here, the code used to assign a new ID for thee officer. I don't now what is it for, but I removed this line to fix this bug.

Reply:
The reason the ID was initially assigned is for the UI to keep officers separate. I think that removing this probably breaks something. Please test well around adding / changing / deleting / saving / resuming.

Reply:
I did more tests around adding / changing / deleting / saving / resuming, all seems good.
However, I found in the filing-template-mixin, this method will remove the parties "id" property after filing
![image](https://github.com/user-attachments/assets/208eb718-4b61-417d-ab9d-c02f4e496bd1)
I added a comment below too. This method is used by other filing types too.
I did tests, although removed "id" doesn't seem to have impact, do you want to keep the "id" for officer? I can create a new method in the filing-template-mixin for restoration only.
(The process is, open an extension application, loading the original officer, who does have an "id" 
- adding a new officer, which has an id assigned; or don't change the officer at all, just file an extension 
- but after filing, the original/new officer's id will be removed, because we called the above method
- after that, open another new extension application, the officer's "id" is no longer here)

Reply:
Alternatively, if we don't want to remove the copy.officer.id = uuidv4(), how about my previous solution
![image](https://github.com/user-attachments/assets/22668599-ca8a-4ca0-9794-6374da24eb4f)

Reply:
I'm wary of your previous solution because, how do we know it's the first (index 0) party you need to undo? What if there are multiple persons in the list?

I think your above solution in filing-template-mixin is OK: only call "prepareParties" if this isn't a draft.

Reply:
Is it ok to keep the current solution here?

Reply:
Yes, this change is OK. I'm still not sure about removing the id generation in the other files.

Reply:
I was thinking about your change in filing-template-mixin, which I believe is OK.

I'm still not sure about removing the id generation here. On the one hand, it seems to work, but on the other hand, I think it must have been put here for a reason. Since other filings using this common component, they need to well tested before I am OK with this change.

Reply:
I'm confused again. I was looking at the screenshot above and thinking that was your current change. It's not. So you didn't change any common code.

However, my concern remains: for restorations, you removed the id assignment. Is it the same code in corrections and why does it work there? Maybe corrections needs more testing to find the same bug?

Reply:
I searched in the code base, it seems corrections didn't use this code (copy.officer.id = uuidv4()).

Reply:
I think you're right about all of this. Let me test a couple of things...

Reply:
Sure, I just updated the filing-template-mixin.ts per your notes below (removed the unused lines)

Reply:
I've finished misc testing, and it looks good! Resolving this conversation.